source/args.c

```diff
@@ -233,7 +233,7 @@ int parse_args(struct args *args, int argc, const char *argv[])
 		{ "runs", required_argument, 0, OPTION_RUNS },
 		{ "threshold", required_argument, 0, OPTION_THRESHOLD },
 		{ "output", required_argument, 0, OPTION_OUTPUT },
-		{ 0 },
+		{ NULL, 0, 0, 0 },
```

Comment:
Which compiler are you using and does this result in a warning/error? If this is problematic, then more files have to be changed, and this should end up being a single commit changing these files: "add proper NULL-termination to arrays".